pkgs/development/libraries/pthread-stubs/default.nix

```diff
@@ -0,0 +1,17 @@
+{stdenv, fetchurl}:
+
+stdenv.mkDerivation rec {
+  name = "pthread-stubs-${version}";
```

Comment:
we already have this package in `pkgs/servers/x11/xorg/default.nix`. It is reachable via `xlibs.libpthreadstubs`